eng/pipelines/client.yml

```diff
@@ -212,21 +208,21 @@ jobs:
           OSVmImage: 'windows-2019'
           JavaVersion: '1.8'
           ProfileFlag: ''
-        Linux - Java LTS:
+        Linux - Java 11:
           OSName: 'Linux'
           OSVmImage: 'ubuntu-16.04'
           JavaVersion: '1.11'
-          ProfileFlag: '-Djava-lts'
-        macOS - Java LTS:
+          ProfileFlag: ''
```

Comment:
We should remove the `ProfileFlag` now that we don't plan to use it any longer.

Reply:
I wondered if this should remain or not. I'll remove it.